cmd/nknd/commands/root.go

```diff
@@ -177,22 +177,29 @@ func nknMain() error {
 		}
 	}
 
+	log.Info("Getting my IP address...")
```

Comment:
shouldn't get IP if user puts IP address in `Hostname` (e.g. node has multiple IP address)